sstable/reader.go

```diff
@@ -37,8 +37,14 @@ const (
 	// as opposed to a hardcoded value.
 	initialReadaheadSize = 64 << 10  /* 64KB */
 	maxReadaheadSize     = 256 << 10 /* 256KB */
+
+	UserPropertyCockroachMVCCTimestampSuffix = "crdb.ts.synthesize"
```

Comment:
One thing I've been debating is if this should be specific to Cockroach MVCC timestamps, as it is here, where the name is "crdb.ts." and the replaced value is a hard-coded representation of a placeholder cockroach mvcc timestamp, or if I should instead try to genericize it: add two props e.g. "pebble.suffix-replace.{from,to}" and then enable rewriting -- to `to` --  if `from` is set, and keep mentions of "timestamp" out?

Reply:
went ahead and did this a a generic "suffix from" to "suffix to"  mapping, with no mention of mvcc or timestamps.